Modules/Synchronization/src/Synchronization.Application/IntegrationEvents/Incoming/RelationshipReactivationCompleted/RelationshipReactivationCompletedIntegrationEvent.cs

```diff
@@ -3,7 +3,7 @@
 namespace Backbone.Modules.Synchronization.Application.IntegrationEvents.Incoming.RelationshipReactivationCompleted;
 public class RelationshipReactivationCompletedIntegrationEvent : IntegrationEvent
 {
-    public RelationshipReactivationCompletedIntegrationEvent(string relationshipId, string peer) : base($"{relationshipId}/Reactivation/Revoked")
+    public RelationshipReactivationCompletedIntegrationEvent(string relationshipId, string peer) : base($"{relationshipId}/Reactivation/Completed")
```

Comment:
We only need the ids for outgoing events.

```suggestion
    public RelationshipReactivationCompletedIntegrationEvent(string relationshipId, string peer)
```

Reply:
Changes made.